index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Tribute Page</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <main id="main">
+      <h1 id="title">Dr. Norman Borlaug</h1>
+      <h6 id="title-bottom">The man who saved a billion lives</h6>
+      <div id="img-div">
+        <img src="tribute-page-main-image.jpg" id="image"/>
+        <div id="img-caption">Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to increase wheat yields - part of his life-long war on hunger.</div>
+      </div>
```

Comment:
### Semantics :

The `img` tag should be enclosed in a `figure` tag instead of `div` tag. Also, the image caption should be in `figcaption` instead of `div`.

Reply:
- [ ] It is important that for every comment you add you should as well go deep by explaining the `why` a change is being requested. This would assist the students to better understand the reason for a change and they know how best to go about it in their next projects :+1: :sparkles: 
- [ ] please i kindly suggest you try to sound polite words when requesting for changes from the student, you can use words such as `i kindly suggest, i think you can, please ensure you` :+1: :sparkles: